source/tutorials/building-smart-contract-only-dapps.md

```diff
@@ -0,0 +1,183 @@
+title: Building Smart Contract only DApps with Embark
+layout: tutorial
+---
+
+Building decentralized applications often involves many parts and components, such as Smart Contracts and a front-end, that have to play well together, in order to provide users the best experience possible. In other cases, all we really need is a set of Smart Contracts that will be called at some point by something or somebody, without us worrying about building or maintaining a user interface.
+
+Embark enables us to implement either of those scenarios and in this article we're going to explore how to build a decentralized applications where Smart Contracts are the primary focus.
+
+## Creating a Smart Contracts only application
+
+Before get started, let's make sure that Embark's command line tool is actually installed. Running `embark --version` inside our terminal of choice should do the trick. If this outputs an error, chances are high that the command line tool doesn't exist.
+
+To change that, all we have to do is using Node's package manager `npm`, using the following command:
+
+```
+$ npm install -g embark@next
```

Comment:
In general I would not put any version directly in article.
@next,
4.0.0

Or I would extract to constants if possible
Otherwise that can become complicated to maintain

Reply:
I fully agree with you. However, since we at the moment "promote" using `@next` in the official docs as well and therefore, we have to go through all the docs anyways to make sure those will be removed onve v4.0 is out, I thought it should be fine.

Also because, I do believe we want people to use v4-beta and this is the only way to make them aware of it.

Happy to remove it, I just think the experience will be less good for the reader when trying this tutorial.

Reply:
Can you elaborate what you mean by constants? Removing `4.0.0` and `3.2.7` from the equation? 

I considered that too, but then I also thought: `@next` is gonna go away with 4.0.0 and then there's only going to be the latest and great (unless you explicitly `@version`). So in other words, with `@next` is very specific to transitioning between 3.x and 4.x.

But I'm happy to remove this and call it "latest developer version" versus "latest stable version" or something like that.

Reply:
Okay I'm leaving this in for now and will update docs+blog once v4 is stable